templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Fine-tuning Llama-3, Mistral and Mixtral with Anyscale\n",
     "\n",
-    "**⏱️ Time to complete**: 2.5 hours for 7/8B models (9 hours for 13B, 25 hours for 70B)\n",
+    "**⏱️ Time to complete**: 60min for 8B model with LoRA, 2.5h with full-parameter fine-tuning\n",
```

Comment:
I've made additional changes to the configs so that we stop at three epochs. That's enough to learn gsm8k as per my previous experiments and certainly enough to demonstrate to people that this thing works.

-> We should steer them away from running endless experiments during their "orientation phase"